vortex/utils/data/loader/__init__.py

```diff
@@ -0,0 +1,38 @@
+supported_loaders = {}
+wrapper_format = {}
+ALL_LOADERS = []
+
+_REQUIRED_ATTRIBUTES = [
+    'create_loader',
+    'supported_loaders'
+]
+
+def register_module(module: str):
```

Comment:
```suggestion
import importlib
import inspect
from typing import Union
from types import ModuleType

def register_module(module: Union[str,ModuleType]):
    """
    register a loader module to vortex registry
    Args:
        module: str or module-type to be registered
    """
```

Reply:
I think it's better if we do not assume that `module` is always in `.`, 
for example, you might be want to import from another module
```
from .my_module import my_loader
register_module(my_loader)
```
but this can't be done since module registering only accepts `str`. Extending this functionality is quite straightforward

see suggestion

Reply:
noted, thanks